util/thread_local_test.cc

```diff
@@ -67,54 +67,53 @@ TEST_F(ThreadLocalTest, UniqueIdTest) {
   port::Mutex mu;
   port::CondVar cv(&mu);
 
-  // perf_context and iostats_context take 2 ids
-  ASSERT_EQ(IDChecker::PeekId(), 2u);
+  ASSERT_EQ(IDChecker::PeekId(), 0u);
```

Comment:
Because previously I changed it to use ThreadLocalPtr there is a global counter summarize the total number of entries in the ptr. If I add iostats and perf_stats there are two more from the start of the db. So for this part, it is just a reversion, nothing special.